src/main/features/core/applicationMenu.js

```diff
@@ -125,7 +125,10 @@ const template = [
       },
       {
         label: 'Close',
-        accelerator: 'CmdOrCtrl+W',
+        accelerator: (() => {
```

Comment:
This creates a new function scope needlessly, why not use the ternery I suggested in the previous comment?